speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/mocks/DecisionMakerMock.h

```diff
@@ -24,8 +24,8 @@ class DecisionMakerMock
         Situation = p_initialSituation;
         Track = p_track;
         BlackboxExecutablePath = p_blackBoxExecutablePath;
-        Recorder = p_recorder;
-        BlackBoxData = p_testSituations;
+        Record = p_recorder;
```

Comment:
Why was this renamed??? I find Recorder to be a clearer name.

Reply:
GCC does not allow class and variable names to be the same thing